canonical/lxd-ui
# Update dependencies to vite 5

## Done

- updated all dependencies
- adjust config
- adjust formatting

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check build and ci passes
    - spot check demo server
